Integrations/integration-ReversingLabsTitaniumCloud.yml

```diff
@@ -0,0 +1,221 @@
+description: 'ReversingLabs Data provides malware status of the sample. Status can
```

Comment:
![image](https://user-images.githubusercontent.com/7270217/33948795-840250c0-e030-11e7-8c03-7986b41a278d.png)
The detailed instructions field is missing. Don't we need one, or it is straight forward to add new instance. If yes then there is no need to fill this field.